src/vision/test/ros-vision-test.cpp

```diff
@@ -1,13 +0,0 @@
-//
-// Created by valerian on 15/10/16.
```

Comment:
we should formalize this

Reply:
I think we already have, is what you mean at the start of "Coding Conventions" in `README.md`?